auth-web/src/components/auth/manage-business/BusinessLookup.vue

```diff
@@ -0,0 +1,179 @@
+import Vue from 'vue'
+import { namespace } from 'vuex-class'
+
+const BusinessModule = namespace('business')
+
+enum States {
+  INITIAL = 'initial',
+  SEARCHING = 'searching',
+  SHOW_RESULTS = 'show results',
+  NO_RESULTS = 'no results',
+  SUMMARY = 'summary'
+}
+
+/*
+ * See PPR's BusinessSearchAutocomplete.vue for a Composition API example.
+ */
+@Component({})
+export default class BusinessLookup extends Vue {
+  // enum for template
+  readonly States = States
+
+  // action from business module
+  @BusinessModule.Action('isAffiliated')
+  private readonly isAffiliated!: (identifier: string) => Promise<boolean>
+
+  /** V-model for search field. */
+  searchField = ''
+
+  /** Results from business lookup API. */
+  searchResults = [] as Array<BusinessLookupResultIF>
+
+  /** State of this component. */
+  state = States.INITIAL
+
+  /** Called when searchField property has changed. */
+  @Watch('searchField')
+  @Debounce(600)
+  private async onSearchFieldChanged (): Promise<void> {
+    // safety check
+    if (this.searchField && this.searchField.length > 2) {
+      this.state = States.SEARCHING
+      const searchStatus = null // search all (ACTIVE + HISTORICAL)
+      this.searchResults = await BusinessLookupServices.search(this.searchField, searchStatus).catch(() => [])
+
+      // enable or disable items according to whether they have already been added
+      for (const result of this.searchResults) {
+        result.disabled = await this.isAffiliated(result.identifier)
+      }
+
+      // display appropriate section
+      this.state = (this.searchResults.length > 0) ? States.SHOW_RESULTS : States.NO_RESULTS
+    } else {
+      // reset variables
+      this.searchResults = []
+      this.state = States.INITIAL
+    }
+  }
+
+  /** When an item has been selected, emits event with business object. */
+  @Emit('business')
+  onItemSelected (input: BusinessLookupResultIF): void {
+    // safety check
+    if (input) {
+      // change to summary state
+      this.state = States.SUMMARY
+    }
+    // event data is automatically returned
+  }
+}
+</script>
+
+<style lang="scss" scoped>
+@import '@/assets/styles/theme.scss';
+
+.font-size-12 {
+  font-size: 12px;
+}
+
+p {
+  color: $gray7;
+}
+
+.business-lookup-result {
+  font-size: $px-14;
+  color: $gray7;
+
+  &:hover {
+    background-color: $gray1;
+    color: $app-blue;
+  }
+}
+
+.result-identifier,
+.result-name {
+  font-size: $px-16;
+
+  // limit col width and show an ellipsis for long names:
+  max-width: 485px;
+  white-space: nowrap;
+  overflow: hidden;
+  text-overflow: ellipsis;
+}
+
+.result-action {
+  text-align: right;
+  font-size: $px-14;
+
+  .select {
+    color: $app-blue;
+  }
+
+  .added {
+    color: $app-green;
+  }
+}
+
+// prevent Magnify icon from being rotated when list is displayed
+::v-deep .v-input__icon .mdi-magnify {
+  -webkit-transform: none !important;
+  transform: none !important;
+}
```

Comment:
Normally, the icon is flipped when the v-menu (list) is shown, which makes sense when the icon is an up triangle or down triangle. It doesn't make sense when the icon is a magnifying glass 🙄